lib/SearchAndSort/SearchAndSort.js

```diff
@@ -533,6 +549,9 @@ class SearchAndSort extends React.Component {
     const sortOrder = orders.slice(0, maxSortKeys).join(',');
 
     this.log('action', `sorted by ${sortOrder}`);
+    // Reset result count when sorting so that only 1 page is initially retrieved
```

Comment:
Are we confident this is the right behavior? For me, the Principle of Least Surprise would be that if I had clicked "Show more" twice and so had 300 records, clicking "Sort" would sort those 300 records. I understand I might get 300 _different_ records if I hadn't "Show more"-ed all the way through the result set, but I just think of sort rearranging the order of things, not changing my limit and offset parameters. 

Maybe this is an unlikely use-case and folks are most likely to sort before clicking "Show more". Dunno; I'm a dev, not a UX guy, so my instincts here are not necessarily useful.

Reply:
Yes, I confirmed with Vince that the result count should reset. The sorting just affects the parameters that are passed into the API calls. So, if we don't reset the count, we potentially need to fire off `n` number of API calls at once depending on how many times the user clicked "load more". Resetting the count is more performant and considered more predictable according to Vince.

Reply:
So sorting will be always limited by the one page size, right?

Reply:
Yes, but this only affects modules that use `searchAndSort`.